src/org/omegat/Bundle.properties

```diff
@@ -1252,6 +1252,9 @@ FILTERMASTER_ERROR_LOADING_FILTER=Filter '{0}' could not be loaded from '{1}'
 
 FILTERMASTER_ERROR_SRC_TRG_SAME_FILE=Translated file path cannot be identical to the source file path:\n{0}
 
+FILTERMASTER_ERROR_UNKNOWN_ENCODING=Detect a wrong character encoding or wrong character in specified encoding when \
+  output to translation file. Please check your project configuration and translations.
+
```

Comment:
How about:
`FILTERMASTER_ERROR_UNKNOWN_ENCODING=Incorrect encoding or unsupported character detected when generating the translated file. Check your source files or filter configuration.`?

Reply:
@miurahr When OmegaT displays the message, does it know the encoding or the character? Maybe displaying the information would be helpful?

Reply:
> The exception is caught at `FilterMaster` where details are not known.

Reply:
OK. Thanks.

In that case, my suggestion remains unchanged.